src/test/java/org/jilt/test/SingularDemoTest.java

```diff
@@ -0,0 +1,30 @@
+package org.jilt.test;
+
+import org.junit.Test;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class SingularDemoTest {
+    @Test
+    public void testSingularAddMethods() {
+        SingularDemoBuilder builder = SingularDemoBuilder.singularDemo();
+        builder.addTag("foo");
+        builder.addTag("bar");
+        builder.addAllTags(Arrays.asList("baz", "qux"));
+        SingularDemo demo = builder.build();
```

Comment:
Can you use a fluent style instead?

```suggestion
        SingularDemo demo = SingularDemoBuilder.singularDemo()
            .addTag("foo");
            .addTag("bar");
            .addAllTags(Arrays.asList("baz", "qux"))
            .build();
```